### PR TITLE
web: Remove all occurences of innerHTML (except in test files)

### LINKED ISSUE #10976: Clean-up and refactoring: Refactoring away usage of setting `innerHTML` in certain places.
In 'web/packages/core/src/ruffle-player.ts' and elsewhere, `innerHTML` is assigned in a number of places. This is most likely not insecure given the usage and the code and how the systems are set up, but it is still a code smell as such, and it would be good to have refactored away.

### PR DESCRIPTION
**Pull request by danielhjacobs**
Fixes #10976

### CODE REVIEW

**Review by n0samu — APPROVED** [review]

File: web/packages/selfhosted/test_assets/swfobject.js
Comment (relrelb) [inline web/packages/selfhosted/test_assets/swfobject.js:1]: I think test files and swfobject.js don't have to be changed.

**Comment by torokati44** [comment]
This is now conflicting due to https://github.com/ruffle-rs/ruffle/pull/12917, which I think also added an `innerHTML` usagee

**Comment by n0samu** [comment]
It didn't add an innerHTML usage but it did change one (not remove it)

File: web/packages/selfhosted/test_assets/swfobject.js
Comment (danielhjacobs) [reply web/packages/selfhosted/test_assets/swfobject.js:1]: Dropped those commits.

**Review by relrelb — APPROVED** [review]

**Pull request merged**